v3/plugins/cdr/code-server/2.1523-vsc1.38.1-openshift/meta.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+publisher: cdr
+name: code-server
+version: 2.1523-vsc1.38.1-openshift
```

Comment:
Could you mention the differences in the `-openshift` version? I foresee some confusion in using this plugin on OpenShift (especially as latest looks like it won't be OpenShift-compatible).

Reply:
OpenShift version just adds OpenShift-specific user ID fixes since OSO has a known security feature that scrambles UIDs, which breaks a lot of containers.

Reply:
Docker containers are built [here](https://github.com/sr229/code-server-openshift) which should give you an idea why openshift is a bit more different.

Reply:
In that case, to smooth user experience, what do you think about using the OpenShift-compatible image in both versions (k8s and OpenShift)? As far as I know, the usual OpenShift fixes aren't incompatible with Kubernetes (and it's possible to configure your kube cluster to do the same arbitrary UID stuff IIRC).

Reply:
Sounds like a plan 👍